.circleci/config.yml

```diff
@@ -603,6 +641,7 @@ workflows:
               only:
                 - master
     jobs:
+      - build_and_test_habitat_lab_conda
```

Comment:
The `build_and_test_habitat_lab_conda` has to be added to `version_conda_release` as well.

Reply:
Well, this PR was pushed months ago. A little late for a review.

Reply:
But it's never late for the fixing PR.

Reply:
> 
> 
> Before this PR
> <img alt="Screen Shot 2021-01-27 at 7 06 22 PM" width="1231" src="https://user-images.githubusercontent.com/877440/106084243-d2be0800-60d2-11eb-975b-37ea15e11424.png">
> After this PR:
> <img alt="Screen Shot 2021-01-27 at 7 05 49 PM" width="1269" src="https://user-images.githubusercontent.com/877440/106084267-e10c2400-60d2-11eb-975a-da1fe3378a2c.png">
> 
> While this PR leveraging parallelization it regresses the CI time for 27%. Will dive deeper to understand what new testing it brings.

That was caused by an orthogonal change to the Habitat-Lab test suite (the addition of more tests for observation transformers) that prompted this change.